client/my-sites/plan-features/header.jsx

```diff
@@ -3,12 +3,14 @@
  */
 import React, { Component, PropTypes } from 'react';
 import noop from 'lodash/noop';
+import identity from 'lodash/identity';
```

Comment:
Is this needed? If you compose the `PlanFeaturesHeader` with `localize` fn, it should always get the `translate` prop. I haven't seen this before, that's why I'm wondering.